openbas-front/src/components/ThemeDark.ts

```diff
@@ -175,10 +175,6 @@ const ThemeDark = (
             color: '#ffffff !important',
             background: `${accent || '#01478d'} !important`,
           },
-          'pre.light': {
-            fontFamily: 'Consolas, monaco, monospace',
-            background: `${nav || '#071a2e'} !important`,
-          },
```

Comment:
Not used anymore ?

Reply:
I couldn't find it in the code and there are two files to manage the themes (there is a pre "light" in ThemeLight.ts)